_pages/contributing/git-flow-and-releases.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Git flow and release process
+---
+
+## Library versions compatibility
+
+In this document we always refer to version as a single instance. However spartacus project is not a single library, but a set of libraries.
+
+To make it easier to know which version of one library is compatible with another library we keep version synchronized across all packages.
+That means when we want to release version `1.5.0` we release all libraries under this version (even if some libraries doesn't have any changes since the last release).
+
+Thanks to that you can be confident that if you install all packages with the same version everything will work correctly together. Different versions of libraries might work well together, but we don't test those configurations and can't promise correct behavior.
+
+## Version support
+
+For versioning we follow semver. (TODO put link here). Apart from `stable` versions we make use of `next` and `rc` releases.
+
+Our assumptions about versions:
+
+- `stable` version is well tested (also by community) spartacus release that will only be patched with bug fixes (versions available under `latest` tag on `npmjs.rg`).
+- `rc` version means that we finished development of all new features for that version (there won't be any major changes in this features and public API) and community can start testing those. These release might contain few bugs that will be fixed before stable release. When there won't be any more bugs and community stops reporting issues for that version we proceed with `stable` release.
+- `next` versions are released when we finish some particular feature so community can test them instantly. This versions can have a lot of bugs. Features and public API might be still a subject for big changes. If you want to test new features as soon as possible this is the versions for you (versions available under `next` tag on `npmjs.org`).
```

Comment:
```suggestion
- `next` versions are released when we finish some particular feature so community can test them instantly. This versions can have a lot of bugs. Features and public API might be still a subject for big changes. If you want to test new features as soon as possible this is the versions for you (versions available under `next` tag on `npmjs.org`). We discourage you from using this version in a production setup, as upgrading from it might be not as easy as between stable versions.
```

Reply:
Note added.